components/treelist.tsx

```diff
@@ -163,7 +163,7 @@ function TreeList({
     <TreeListWrapper $containerStyle={containerStyle}>
       {actions && (
         <ActionsWrapper>
-          {actions.map((data, index) => {
+          {actions.map((props, index) => {
```

Comment:
As much as possible, I believe it's better that, for example, if the variable name is `actions` then the individual object being iterated/mapped is named `action`, so:

```js
          {actions.map((action, index) => {
```

Sometimes, this kind of style is not possible, but if possible and reads better, please do so :)